router/config.go

```diff
@@ -668,6 +671,13 @@ func LoadConfigWithOptions(path string, loadIdentity bool) (*Config, error) {
 					return nil, errors.New("invalid value for metrics.messageQueueSize")
 				}
 			}
+			if value, found := submap["eventQueueSize"]; found {
+				if intVal, ok := value.(int); ok {
+					cfg.Metrics.EventQueueSize = intVal
```

Comment:
makes me think the minimum size (16) actually belongs here instead of in the metrics library

Reply:
Not going to worry about it for now. Setting it to a low value should have minimal effect on a system, and if someone goes to the effort to set it, they have a reason for it.